README.md

```diff
@@ -8,13 +8,22 @@ It uses [Zola](https://www.getzola.org/) and is deployed to GitHub Pages via Git
 
 ## Building
 
-To serve the site locally, first install Zola: (takes a couple minutes)
+To serve the site locally, first install Zola:
+
+Linux, MacOS:
+
+```sh
+curl --proto '=https' --tlsv1.2 -LsSf https://github.com/senekor/zola/releases/download/v0.20.1-senekor/zola-installer.sh | sh
+```
+
+Windows:
 
 ```sh
-# using a fork because we rely on a few patches that haven't landed yet
-cargo install --locked --git https://github.com/senekor/zola --rev 79410eea82f837e4de9b1e4c3905287060b69255
+powershell -ExecutionPolicy Bypass -c "irm https://github.com/senekor/zola/releases/download/v0.20.1-senekor/zola-installer.ps1 | iex"
```

Comment:
still don't want to tell people to run a script unless it's Rust-hosted.

Reply:
Oh, sorry I misread. I thought you said the script is ok for devs but not CI. I guess then we can scrap this PR?

What happens when Zola has a new official release? Do we still tell blog authors to build Zola from source instead of downloading the binaries?

Reply:
Yes, I think we should have people `cargo install` it, with a pinned hash.

Reply:
Got it. Zola isn't published on crates.io, so an official release won't actually matter to us anymore.

Reply:
An alternative is to do it all locally with a submodule or a crate that depends on it. Then it'd deterministic.

Reply:
I'm not sure I'm following. A crate that depends on a git version of zola? What's the advantage of that? Also, there's no separation of `main.rs` and `lib.rs` in Zola. It doesn't look like it's supposed to be used as a library.

Reply:
Ah, and the advantage is that when we update Zola in the future, it will automatically be picked up and we don't have to worry about users running outdated versions of Zola?

Reply:
Correct.